obs/obs-api.yaml

```
@@ -156,6 +156,33 @@ paths:
                 format: binary
         default:
           $ref: "#/components/responses/ErrorResponse"
+  /files/{id}/internal:
+    get:
+      tags:
+        - Files
+      summary: Get contnet refernce data of the file
```

```suggestion
      summary: Get content reference data of the file
```
?
but anyway I do not understand the summary :D